Components/ReusableComponent/admin.js

```diff
@@ -0,0 +1,20 @@
+/* globals acf */
+import $ from 'jquery'
+
+acf.addAction('select2_init', function ($select, args, settings, field) {
+  const $fieldEl = field.$el
+
+  $select.on('change', { $fieldEl: $fieldEl }, function () {
+    const postId = $(this).val()
+    const postTitle = $(this).find('option:selected').text()
+    const postLink = `/wp/wp-admin/post.php?post=${postId}&action=edit&classic-editor`
```

Comment:
I think the admin link could be different based on the installation. The path `wp` could be different. I assume this can be fixed by using WordPress path variables or functions.

Reply:
I didn't know that thanks will look into it and refactor!